### DISCUSSION TITLE
Binaries in Linux bundle since v6.0.0

### DISCUSSION THREAD

**Discussion by lucendio** [concourse/concourse#6285] [Help & Support]
Hi,

it all started with [gdn assets not getting updated](https://github.com/concourse/concourse/issues/6236) when updating Concourse to `6.7.1`. During installation, the assets are being copied over to `/var/gdn/assets`. But instead of copying them from the `bin` directory of the archive, they are bundled within `gdn` and extracted from there.

I don't want to argue about the approach. I'm rather interested in as to why these binaries are put into the `bin` directory, especially in context of deploying Concourse? The documentation is a little sparse around that topic.

**Comment by taylorsilva** [comment]
If I'm understanding your question correctly then I think this is a question for the [guardian team](https://github.com/cloudfoundry/guardian/) since it is their binary that extracts into the `/var/gdn/assets` directory.

**Reply by lucendio** [reply]
Not exactly I think. It is rather the other way around. Why are the binaries placed the archive under `./bin` if they are already bundled into the `gdn` binary (or at least some of them)?

**Reply by taylorsilva** [reply]
ah I see now!
```
$ find / -name 'runc'
/usr/local/concourse/bin/runc
/var/gdn/assets/linux/bin/runc
```

I didn't realize it was under there!

Looked at some dockerfiles and I think it's for containerd which would explain why you only start seeing in v6: https://github.com/concourse/ci/blob/86770a0d770e9d73b59534fec1b3e9312e466024/dockerfiles/dev/Dockerfile#L23-L35
